configure.ac

```diff
@@ -1,13 +1,13 @@
 # Process this file with autoconf to produce a configure script.
 
-AC_INIT([meep],[m4_esyscmd(./version.sh 1.29.0-beta)])
+AC_INIT([meep],[m4_esyscmd(./version.sh 1.29.0)])
 AC_CONFIG_SRCDIR(src/step.cpp)
 
 # Shared-library version number; indicates api compatibility, and is
 # not the same as the "public" version number.  (Don't worry about this
 # except for public releases.) Note that any change to a C++ class
 # definition (in the .hpp file) generally breaks binary compatibility.
-SHARED_VERSION_INFO="32:1:0" # CURRENT:REVISION:AGE
+SHARED_VERSION_INFO="33:0:0" # CURRENT:REVISION:AGE
```

Comment:
Note that BFAST broke binary compatibility so bumping the major version is correct here.